## 86% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 3
Expected to fail tests: 4
Passed tests: 44
```
```
IDs of failed tests: 727230,703503,727229 
```
```
IDs of expected to fail tests: 703495,702948,727232,703629 
```
<h3>Failed tests (3)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=20253-remove-non-operable-accounts-in-switcher&PR_ID=20457&APK_NAME=20457.apk&TR_CASE_IDS=727230,703503,727229">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579229">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div># STEP: Check Status community tags in the Discover communities screen</div><div># STEP: Check Status Inu community tags in the Discover communities screen</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:300: in test_community_discovery
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community ' 0xUX' is not in the Discover Communities list.
E    Community 'Status' is not in the Discover Communities list.
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b4040bcaa3d47d9af99686e38dbae58?auth=79b2f0884b092508432159265909a7df#355">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579234">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div># STEP: Getting ETH amount in the wallet of the sender before transaction</div><div>Device 1: Find `WalletTab` by `accessibility id`: `wallet-stack-tab` </div></blockquote></p>

```
critical/test_wallet.py:120: in test_wallet_send_asset_from_drawer
    sender_balance, receiver_balance, eth_amount_sender, eth_amount_receiver = self._get_balances_before_tx()
critical/test_wallet.py:41: in _get_balances_before_tx
    self.wallet_1.wallet_tab.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: WalletTab by accessibility id: `wallet-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579233">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
critical/test_wallet.py:111: in test_wallet_send_eth
    self.wallet_1.send_asset(address=self.receiver['address'], asset_name='Ether', amount=amount_to_send)
../views/wallet_view.py:100: in send_asset
    self.confirm_transaction()
../views/wallet_view.py:87: in confirm_transaction
    self.slide_and_confirm_with_password()
../views/wallet_view.py:81: in slide_and_confirm_with_password
    self.slide_button_track.slide()
../views/base_view.py:257: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:308: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:294: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table></details><h3>Expected to fail tests (4)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579217">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'#cats')]` </div><div>Device 2: Looking for a message by text: it is just a message text </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1124: in test_community_hashtag_links_to_community_channels
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32a789c0bd4441c28e2dae8ff36e951a?auth=011458ec9e1bbd473f929e28bbfdaa5d#304">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d08c1f59f6b44d67a981c5ef885c6556?auth=be04473b71db27d7ee2d9f7fc2287850#293">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579230">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><p><blockquote><div>Device 2: Looking for community: 'open community' </div><div>Device 2: Click until `Text` by `accessibility id`: `community-description-text` will be presented </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1212: in test_community_join_when_node_owner_offline
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join, it was indicated as not sent for the sender before he went offline 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32a789c0bd4441c28e2dae8ff36e951a?auth=011458ec9e1bbd473f929e28bbfdaa5d#503">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d08c1f59f6b44d67a981c5ef885c6556?auth=be04473b71db27d7ee2d9f7fc2287850#475">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579227">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div># STEP: Change device time so chat will be unmuted by timer</div><div>Device 2: Long press on `ChatElement` </div></blockquote></p>

```
critical/chats/test_group_chat.py:497: in test_group_chat_mute_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Chat is still muted after timeout 
```

<b>[[Chat is not unmuted after expected time: https://github.com/status-im/status-mobile/issues/19627]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cd87f70ed439419681fe4a6566e11b95?auth=bc38ce5820472f5b1a1c4044c171f4f9#474">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d24ae5ef1984b30b09f93c2b3563802?auth=8fbe15ab00ee6a941a85108f9acc43ba#276">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1f0b2a5aa3a46d09beb5f274524d190?auth=362eb94f5e24e80716d7a02cf983cc63#243">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579236">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: `Text` is `0x8d2413447ff297d30bdc475f6d5cb00254685aae` </div><div>Device 1: Click system back button </div></blockquote></p>

```
critical/test_wallet.py:214: in test_wallet_add_remove_watch_only_account
    self.home_view.driver.fail(
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Incorrect address '0x8d2413447ff297d30bdc475f6d5cb00254685aae' is shown when swiping between accounts, expected one is '0x8d2413447ff297d30bdc475f6d5cb00254685aae' 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3671ce8ed49b4d0c92904ddc4756c1de?auth=527129e23feda7278193af974f978de3#103">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (44)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579200">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32a789c0bd4441c28e2dae8ff36e951a?auth=011458ec9e1bbd473f929e28bbfdaa5d#185">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d08c1f59f6b44d67a981c5ef885c6556?auth=be04473b71db27d7ee2d9f7fc2287850#169">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579197">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32a789c0bd4441c28e2dae8ff36e951a?auth=011458ec9e1bbd473f929e28bbfdaa5d#136">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d08c1f59f6b44d67a981c5ef885c6556?auth=be04473b71db27d7ee2d9f7fc2287850#111">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579208">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32a789c0bd4441c28e2dae8ff36e951a?auth=011458ec9e1bbd473f929e28bbfdaa5d#301">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d08c1f59f6b44d67a981c5ef885c6556?auth=be04473b71db27d7ee2d9f7fc2287850#271">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579196">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0439040a8603460b88aec6dcd0f4c98d?auth=86cdb1fb683426fc8be108597a975519#172">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04c733d72aa64c35a8d0a6ad2a829657?auth=043f4b595193ea5282613d91756310b1#126">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579195">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0439040a8603460b88aec6dcd0f4c98d?auth=86cdb1fb683426fc8be108597a975519#96">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04c733d72aa64c35a8d0a6ad2a829657?auth=043f4b595193ea5282613d91756310b1#85">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579228">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0439040a8603460b88aec6dcd0f4c98d?auth=86cdb1fb683426fc8be108597a975519#108">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/04c733d72aa64c35a8d0a6ad2a829657?auth=043f4b595193ea5282613d91756310b1#113">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579235">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3671ce8ed49b4d0c92904ddc4756c1de?auth=527129e23feda7278193af974f978de3#39">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579216">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c56de1e97f484fcc9124dcfa959ca775?auth=80931f94c1d162533f8e4b20c8720ead#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bc0272d99d3c479e97893c1b57dc09f2?auth=eee62bdbf244773848c18fb9e3feec15#99">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579218">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/55af431995274fb183320f94f409dc31?auth=9deb1b86e807824832ab5f0b5a506782#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce965ddd2f154984895a7dae4370c031?auth=7e96c6bcd1385ce31ad624cadc2766bf#101">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579219">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/55af431995274fb183320f94f409dc31?auth=9deb1b86e807824832ab5f0b5a506782#170">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce965ddd2f154984895a7dae4370c031?auth=7e96c6bcd1385ce31ad624cadc2766bf#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579231">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86f97775bd854c6c9eff594b49fde399?auth=68b3f40f4238a240e57155171d1cc699#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579192">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86f97775bd854c6c9eff594b49fde399?auth=68b3f40f4238a240e57155171d1cc699#204">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579221">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b4040bcaa3d47d9af99686e38dbae58?auth=79b2f0884b092508432159265909a7df#230">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579190">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b4040bcaa3d47d9af99686e38dbae58?auth=79b2f0884b092508432159265909a7df#77">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579214">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b4040bcaa3d47d9af99686e38dbae58?auth=79b2f0884b092508432159265909a7df#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579209">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b4040bcaa3d47d9af99686e38dbae58?auth=79b2f0884b092508432159265909a7df#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579225">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b4040bcaa3d47d9af99686e38dbae58?auth=79b2f0884b092508432159265909a7df#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579193">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2030e3e9d5484a7691872d28bbefa4f4?auth=70ec01355b0786e8a0c54d4f532f5c45#135">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/460cb822907248009754fddd27f001a5?auth=70bf907ca10f03fd00683535835c31fa#220">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579211">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2030e3e9d5484a7691872d28bbefa4f4?auth=70ec01355b0786e8a0c54d4f532f5c45#100">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/460cb822907248009754fddd27f001a5?auth=70bf907ca10f03fd00683535835c31fa#122">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579210">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2030e3e9d5484a7691872d28bbefa4f4?auth=70ec01355b0786e8a0c54d4f532f5c45#57">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/460cb822907248009754fddd27f001a5?auth=70bf907ca10f03fd00683535835c31fa#56">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579194">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6271a449beb34b109ef1ac86b573e4f7?auth=f39e46a085ab3e3a6411606000271982#148">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0de095af59c64e37a457961a4483f7f2?auth=77fe1614d8e6d3d344f5a7be60892b56#120">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579189">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6271a449beb34b109ef1ac86b573e4f7?auth=f39e46a085ab3e3a6411606000271982#555">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0de095af59c64e37a457961a4483f7f2?auth=77fe1614d8e6d3d344f5a7be60892b56#401">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579201">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6271a449beb34b109ef1ac86b573e4f7?auth=f39e46a085ab3e3a6411606000271982#414">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0de095af59c64e37a457961a4483f7f2?auth=77fe1614d8e6d3d344f5a7be60892b56#260">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579191">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6271a449beb34b109ef1ac86b573e4f7?auth=f39e46a085ab3e3a6411606000271982#356">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0de095af59c64e37a457961a4483f7f2?auth=77fe1614d8e6d3d344f5a7be60892b56#210">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579212">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6271a449beb34b109ef1ac86b573e4f7?auth=f39e46a085ab3e3a6411606000271982#463">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0de095af59c64e37a457961a4483f7f2?auth=77fe1614d8e6d3d344f5a7be60892b56#291">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579226">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6271a449beb34b109ef1ac86b573e4f7?auth=f39e46a085ab3e3a6411606000271982#494">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0de095af59c64e37a457961a4483f7f2?auth=77fe1614d8e6d3d344f5a7be60892b56#341">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579187">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6271a449beb34b109ef1ac86b573e4f7?auth=f39e46a085ab3e3a6411606000271982#210">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0de095af59c64e37a457961a4483f7f2?auth=77fe1614d8e6d3d344f5a7be60892b56#165">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579186">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6271a449beb34b109ef1ac86b573e4f7?auth=f39e46a085ab3e3a6411606000271982#96">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0de095af59c64e37a457961a4483f7f2?auth=77fe1614d8e6d3d344f5a7be60892b56#85">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579222">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/41d90404c8ec41e79cd31faf6c6929b7?auth=6cbdc6477c0a3c020ca5412c68d19196#234">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5d4859139fa4ac1856e63224656d119?auth=b5ec27cef2d4f87949b8e5492c4ee722#165">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579213">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/41d90404c8ec41e79cd31faf6c6929b7?auth=6cbdc6477c0a3c020ca5412c68d19196#259">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5d4859139fa4ac1856e63224656d119?auth=b5ec27cef2d4f87949b8e5492c4ee722#226">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579204">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/41d90404c8ec41e79cd31faf6c6929b7?auth=6cbdc6477c0a3c020ca5412c68d19196#307">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5d4859139fa4ac1856e63224656d119?auth=b5ec27cef2d4f87949b8e5492c4ee722#286">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579220">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/41d90404c8ec41e79cd31faf6c6929b7?auth=6cbdc6477c0a3c020ca5412c68d19196#553">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5d4859139fa4ac1856e63224656d119?auth=b5ec27cef2d4f87949b8e5492c4ee722#472">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579215">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/41d90404c8ec41e79cd31faf6c6929b7?auth=6cbdc6477c0a3c020ca5412c68d19196#426">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5d4859139fa4ac1856e63224656d119?auth=b5ec27cef2d4f87949b8e5492c4ee722#394">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579232">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/41d90404c8ec41e79cd31faf6c6929b7?auth=6cbdc6477c0a3c020ca5412c68d19196#588">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5d4859139fa4ac1856e63224656d119?auth=b5ec27cef2d4f87949b8e5492c4ee722#498">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579203">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/41d90404c8ec41e79cd31faf6c6929b7?auth=6cbdc6477c0a3c020ca5412c68d19196#201">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5d4859139fa4ac1856e63224656d119?auth=b5ec27cef2d4f87949b8e5492c4ee722#136">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579202">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/41d90404c8ec41e79cd31faf6c6929b7?auth=6cbdc6477c0a3c020ca5412c68d19196#159">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5d4859139fa4ac1856e63224656d119?auth=b5ec27cef2d4f87949b8e5492c4ee722#107">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579207">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/41d90404c8ec41e79cd31faf6c6929b7?auth=6cbdc6477c0a3c020ca5412c68d19196#331">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5d4859139fa4ac1856e63224656d119?auth=b5ec27cef2d4f87949b8e5492c4ee722#303">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579206">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/41d90404c8ec41e79cd31faf6c6929b7?auth=6cbdc6477c0a3c020ca5412c68d19196#177">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5d4859139fa4ac1856e63224656d119?auth=b5ec27cef2d4f87949b8e5492c4ee722#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579205">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/41d90404c8ec41e79cd31faf6c6929b7?auth=6cbdc6477c0a3c020ca5412c68d19196#396">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5d4859139fa4ac1856e63224656d119?auth=b5ec27cef2d4f87949b8e5492c4ee722#364">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579188">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cd87f70ed439419681fe4a6566e11b95?auth=bc38ce5820472f5b1a1c4044c171f4f9#354">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d24ae5ef1984b30b09f93c2b3563802?auth=8fbe15ab00ee6a941a85108f9acc43ba#233">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1f0b2a5aa3a46d09beb5f274524d190?auth=362eb94f5e24e80716d7a02cf983cc63#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579224">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cd87f70ed439419681fe4a6566e11b95?auth=bc38ce5820472f5b1a1c4044c171f4f9#281">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d24ae5ef1984b30b09f93c2b3563802?auth=8fbe15ab00ee6a941a85108f9acc43ba#171">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1f0b2a5aa3a46d09beb5f274524d190?auth=362eb94f5e24e80716d7a02cf983cc63#142">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579223">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cd87f70ed439419681fe4a6566e11b95?auth=bc38ce5820472f5b1a1c4044c171f4f9#174">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d24ae5ef1984b30b09f93c2b3563802?auth=8fbe15ab00ee6a941a85108f9acc43ba#107">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1f0b2a5aa3a46d09beb5f274524d190?auth=362eb94f5e24e80716d7a02cf983cc63#87">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579198">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cd87f70ed439419681fe4a6566e11b95?auth=bc38ce5820472f5b1a1c4044c171f4f9#157">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d24ae5ef1984b30b09f93c2b3563802?auth=8fbe15ab00ee6a941a85108f9acc43ba#79">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1f0b2a5aa3a46d09beb5f274524d190?auth=362eb94f5e24e80716d7a02cf983cc63#79">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3579199">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cd87f70ed439419681fe4a6566e11b95?auth=bc38ce5820472f5b1a1c4044c171f4f9#321">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d24ae5ef1984b30b09f93c2b3563802?auth=8fbe15ab00ee6a941a85108f9acc43ba#206">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d1f0b2a5aa3a46d09beb5f274524d190?auth=362eb94f5e24e80716d7a02cf983cc63#202">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/contexts/wallet/send/events.cljs
+++ b/src/status_im/contexts/wallet/send/events.cljs
@@ -210,12 +210,10 @@
             :always      (assoc-in [:wallet :ui :send :token-display-name]
                           (:symbol token))
             :always      (assoc-in
-                          [:wallet :ui :send
-                           :token-not-supported-in-receiver-networks?]
+                          [:wallet :ui :send :token-not-supported-in-receiver-networks?]
                           token-not-supported-in-receiver-networks?)
             token        (assoc-in [:wallet :ui :send :token] token)
-            token-symbol (assoc-in [:wallet :ui :send :token-symbol]
-                          token-symbol))
+            token-symbol (assoc-in [:wallet :ui :send :token-symbol] token-symbol))
       :fx [[:dispatch [:wallet/clean-suggested-routes]]
            [:dispatch
             [:wallet/wizard-navigate-forward
@@ -253,8 +251,7 @@
                      :collectible
                      :token-display-name
                      :amount
-                     (when (send-utils/tx-type-collectible?
-                            transaction-type)
+                     (when (send-utils/tx-type-collectible? transaction-type)
                        :tx-type))})))
 
 (rf/reg-event-fx

--- a/src/status_im/contexts/wallet/sheets/account_options/view.cljs
+++ b/src/status_im/contexts/wallet/sheets/account_options/view.cljs
@@ -108,8 +108,9 @@
   []
   (let [options-height (reagent/atom 0)]
     (fn []
-      (let [theme                  (quo.theme/use-theme)
-            accounts               (rf/sub [:wallet/accounts-without-current-viewing-account])
+      (let [theme (quo.theme/use-theme)
+            accounts (rf/sub
+                      [:wallet/fully-or-partially-operable-accounts-without-current-viewing-account])
             show-account-selector? (pos? (count accounts))]
         [:<>
          (when show-account-selector?

--- a/src/status_im/contexts/wallet/sheets/select_account/view.cljs
+++ b/src/status_im/contexts/wallet/sheets/select_account/view.cljs
@@ -20,7 +20,7 @@
 (defn view
   []
   (let [selected-account-address (rf/sub [:wallet/current-viewing-account-address])
-        accounts                 (rf/sub [:wallet/accounts-without-watched-accounts])]
+        accounts (rf/sub [:wallet/fully-or-partially-operable-accounts-without-watched-accounts])]
     [:<>
      [quo/drawer-top {:title (i18n/label :t/select-account)}]
      [gesture/flat-list

--- a/src/status_im/subs/wallet/wallet.cljs
+++ b/src/status_im/subs/wallet/wallet.cljs
@@ -446,6 +446,22 @@
  (fn [accounts]
    (remove :watch-only? accounts)))
 
+(defn- keep-fully-or-partially-operable-accounts
+  [accounts]
+  (filter (fn fully-or-partially-operable? [{:keys [operable]}]
+            (#{:fully :partially} operable))
+          accounts))
+
+(rf/reg-sub
+ :wallet/fully-or-partially-operable-accounts-without-current-viewing-account
+ :<- [:wallet/accounts-without-current-viewing-account]
+ keep-fully-or-partially-operable-accounts)
+
+(rf/reg-sub
+ :wallet/fully-or-partially-operable-accounts-without-watched-accounts
+ :<- [:wallet/accounts-without-watched-accounts]
+ keep-fully-or-partially-operable-accounts)
+
 (rf/reg-sub
  :wallet/accounts-with-current-asset
  :<- [:wallet/accounts-without-watched-accounts]

--- a/src/status_im/subs/wallet/wallet.cljs
+++ b/src/status_im/subs/wallet/wallet.cljs
@@ -464,7 +464,7 @@
 
 (rf/reg-sub
  :wallet/accounts-with-current-asset
- :<- [:wallet/accounts-without-watched-accounts]
+ :<- [:wallet/fully-or-partially-operable-accounts-without-watched-accounts]
  :<- [:wallet/wallet-send-token-symbol]
  :<- [:wallet/wallet-send-token]
  (fn [[accounts token-symbol token]]

--- a/src/status_im/subs/wallet/wallet_test.cljs
+++ b/src/status_im/subs/wallet/wallet_test.cljs
@@ -15,10 +15,12 @@
 (def ^:private accounts-with-tokens
   {:0x1 {:tokens                    [{:symbol "ETH"} {:symbol "SNT"}]
          :network-preferences-names #{}
-         :customization-color       nil}
+         :customization-color       nil
+         :operable                  :fully}
    :0x2 {:tokens                    [{:symbol "SNT"}]
          :network-preferences-names #{}
-         :customization-color       nil}})
+         :customization-color       nil
+         :operable                  :partially}})
 
 (def tokens-0x1
   [{:decimals                   1
@@ -481,7 +483,8 @@
       (is (match? result
                   [{:tokens                    [{:symbol "ETH"} {:symbol "SNT"}]
                     :network-preferences-names #{}
-                    :customization-color       nil}]))))
+                    :customization-color       nil
+                    :operable                  :fully}]))))
 
   (testing "returns the accounts list with the current asset using token"
     (swap! rf-db/app-db
@@ -492,7 +495,8 @@
       (is (match? result
                   [{:tokens                    [{:symbol "ETH"} {:symbol "SNT"}]
                     :network-preferences-names #{}
-                    :customization-color       nil}]))))
+                    :customization-color       nil
+                    :operable                  :fully}]))))
 
   (testing
     "returns the full accounts list with the current asset using token-symbol if each account has the asset"

